compiler/test/fail_compilation/templateoverload.d

```diff
@@ -0,0 +1,22 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/templateoverload.d(17): Error: template `templateoverload.T` does not match any template declaration
```

Comment:
Shouldn't this be

```
`template instance templateoverload.T!(1)` does not match any template declaration
```

?

Reply:
Thanks, I made https://github.com/dlang/dmd/pull/14604

Reply:
Awesome!